pkg/controller/migmigration/backup.go

```diff
@@ -121,7 +142,7 @@ func (t *Task) ensureStageBackup() (*velero.Backup, error) {
 	newBackup.Labels[migapi.MigPlanLabel] = string(t.PlanResources.MigPlan.UID)
 	var includedResources mapset.Set
 
-	if t.indirectImageMigration() || Settings.DisImgCopy {
+	if t.indirectImageMigration() || Settings.DisImgCopy || !t.Owner.IsStateMigration() {
```

Comment:
This logic seems wrong. Before the change, we exclude imagestreams in stage backupif we're doing DIM *or* we're disabling image copy -- we don't want imagestreams for DIM because they're handled by the DIM controller, unless we're disabling image copy, in which case we need them to create the placeholder (empty) imagestream. With this change we'll always include imagestreams in stage backup for non-state-migration migrations, including regular DIM migrations where they should be included (or we'll be doing both direct and indirect image copy).

Reply:
@sseago Thanks for pointing that out... looks like we should be ANDing instead of ORing

Reply:
@pranavgaikwad what is the intended behavior here -- i.e. how should a stage backup for state transfer differ from a stage backup for a regular migration? how should an initial backup for state transfer differ from an initial backup for state transfer?

Reply:
@sseago we just want to exclude images from the Stage Backup